core/src/test/scripts/test_data/study_es_0/data_gene_panel_matrix.txt

```diff
@@ -0,0 +1,2 @@
+SAMPLE_ID	study_es_0_mutations
+TCGA-A1-A0SB-01	example_gene_panel_stable_id
```

Comment:
I think it should at least be included in the automated test, and added to `cbioportal_common.META_FIELD_MAP`. The validator warns about keys in meta_* files that it does not recognise at all, so that you notice if, say, a file is being recognised as the wrong data type or there's a typo in one of the keys. Or are a matrix file and a property (and clinical attribute) not supported in the same study?